src/Aquifer.Public.API/Program.cs

```diff
@@ -17,10 +17,11 @@
     ?? throw new InvalidOperationException($"Unable to bind {nameof(ConfigurationOptions)}.");
 
 builder.Services
-    .AddDbContext<AquiferDbContext>(options => options
-        .UseAzureSql(configuration.ConnectionStrings.BiblioNexusDb, providerOptions => providerOptions.EnableRetryOnFailure(3))
+    .AddDbContext<AquiferDbReadOnlyContext>(options => options
+        .UseAzureSql(configuration.ConnectionStrings.BiblioNexusReadOnlyDb, providerOptions => providerOptions.EnableRetryOnFailure(3))
         .EnableSensitiveDataLogging(sensitiveDataLoggingEnabled: builder.Environment.IsDevelopment())
         .UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking))
+    .AddScoped<AquiferDbContext, AquiferDbReadOnlyContext>()
```

Comment:
This enables common services that still use `AquiferDbContext` to inject the new `AquiferDbReadOnlyContext` instead for the Public API.